credentials/utopia-natcert/credential.json

```diff
@@ -65,14 +66,7 @@
         "addressCountry": "Utopia"
       },
       "naturalizedBy": "Utopian High Court for the Center District of Utopia",
-      "naturalizedOn": "2019-10-17"
-    },
-    "confidenceMethod": {
```

Comment:
As discussed elsewhere, this should remain.

Reply:
Fixed in [3f0d5d5](https://github.com/credential-handler/vc-examples/pull/76/commits/3f0d5d5fa7fa2f3307c79a2d0c54f456f92ce870)